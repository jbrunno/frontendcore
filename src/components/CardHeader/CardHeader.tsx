import React from 'react';
import {
  CardHeader as MuiCardHeader,
  CardHeaderProps as MuiCardHeaderProps,
} from '@mui/material';

export type CardHeaderProps = MuiCardHeaderProps;

export function CardHeader(props: CardHeaderProps) {
  return <MuiCardHeader {...props} />;
}

export default CardHeader;
