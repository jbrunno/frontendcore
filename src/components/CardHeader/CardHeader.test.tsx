import React from 'react';
import { render } from '@test-utils/component-testing';
import { CardHeader } from './CardHeader';

test('renders the CardHeader', () => {
  const { container } = render(<CardHeader />);
  expect(container).toMatchSnapshot();
});
