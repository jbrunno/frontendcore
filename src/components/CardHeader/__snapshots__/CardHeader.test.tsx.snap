// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CardHeader 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 16px;
}

.emotion-1 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
}

<div>
  <div
    class="MuiCardHeader-root emotion-0"
  >
    <div
      class="MuiCardHeader-content emotion-1"
    />
  </div>
</div>
`;
